packages/designer/src/components/editor/store.tsx

```diff
@@ -1401,7 +1401,7 @@ export const components: Record<string, Component> = {
 					modalElement = element;
 				}}
 			>
-				<KolCard _heading="Ich bin ein Modal">
+				<KolCard _heading="Ich bin ein Modal" style="width: 100%">
```

Comment:
Die Card sollte sich zu 100% in der Höhe und Breite an den umliegenden Container anpassen. Generell. Basis-CSS?